Documentation/configuration/api-restrictions-table.rst

```diff
@@ -72,6 +72,7 @@ GetMapNameEvents      -
 GetNodeIds            Retrieves a list of node IDs allocated by the agent and
                       their associated node IP addresses.
 GetPolicy             Returns the entire policy tree with all children.
+                      Deprecated: will be removed in v1.19
```

Comment:
Maybe the `get` is still useful to keep just for debugging? Or is it being marked as deprecated because it'll get removed and replaced by an equivalent statedb-based command to dump policies?